pkg/webhook/server.go

```diff
@@ -63,6 +63,7 @@ func NewServer(name, namespace string, port int, cm *certificate.Manager, handle
 // Run actually starts the server.
 func (s *Server) Run(ctx context.Context) error {
 	if err := s.configureCertificateRotation(ctx); err != nil {
+		log.Error().Err(err).Msg("failed to configure cert rotation")
```

Comment:
prefer to log or return errors, not both (we don't do a good job of this in the code base, but can avoid adding new ones).